config.json

```diff
@@ -1,4 +1,7 @@
 {
+  "general": {
+    "telemetry_buffer_length": 20
```

Comment:
I'd prefer you implement this as just a separate field in the config, instead of inside the sub-key `general`. It's not very descriptive and I think it would be easier to parse that way.